pkgs/applications/graphics/solvespace/default.nix

```diff
@@ -42,7 +42,7 @@ stdenv.mkDerivation rec {
 
   meta = with lib; {
     description = "A parametric 3d CAD program";
-    license = licenses.gpl3;
+    license = licenses.gpl3Only;
```

Comment:
https://github.com/solvespace/solvespace/issues/943

Reply:
Oh, must've misread; the `COPYING.md` indeed says later